### PR TITLE
uchv-63 - added where did the incident take place step

### PR DESCRIPTION
**Pull request by gxxm**
* added the routes and forks in steps config
* added fields to fields config
* added validations
* added translations for fields and validation
* added acceptance tests to this step

### CODE REVIEW

**Comment by bendiggle** [comment]
😍 😍 😍 😍 😍 😍 😍

**Comment by bendiggle** [comment]
That's @joefitter for LGTM!

File: apps/complaints/acceptance/pages/where.js [hunk 1]
```diff
@@ -1,5 +1,12 @@
 'use strict';
 
 module.exports = {
-  url: 'where'
+  url: 'where',
+  id: {
+    'where': '#where-group',
+    'phone': '#where-phone',
+    'vac': '#where-vac',
```
Comment (joefitter) [inline apps/complaints/acceptance/pages/where.js]: what is vac? Also these keys don't need `''`

File: apps/complaints/acceptance/pages/where.js [hunk 2]
```diff
@@ -1,5 +1,12 @@
...
+    'psc': '#where-psc',
```
Comment (joefitter) [inline apps/complaints/acceptance/pages/where.js]: what is psc?

File: apps/complaints/index.js [hunk 3]
```diff
@@ -75,6 +75,27 @@ module.exports = {
       }
     },
     '/where': {
+      next: '/phone',
+      forks: [{
+        target: '/vac',
```
Comment (joefitter) [inline apps/complaints/index.js]: is vac the best name for a user facing url?

File: apps/complaints/index.js [hunk 4]
```diff
@@ -75,6 +75,27 @@ module.exports = {
...
+        condition: {
+          field: 'where',
+          value: 'vac'
+        }
+      }, {
+        target: '/psc',
```
Comment (joefitter) [inline apps/complaints/index.js]: as above

**Comment by joefitter** [comment]
👍

**Pull request merged**